CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- JSON API: The `pay` command now accepts a new parameter, `fuzzpercent` used to fuzz the fees for route calculation.
```

Comment:
I've been looking at this metric, and it doesn't really make sense.  Fees are so tiny, that 5% is hardly a change at all, and seems to contribute almost nothing to route diversity :(   But we don't want to increase it so much that we breach 'maxfeepercent' which people really care about!

I'd be happier with moving the routing algo towards randomizing the route so that it spends up to min(maxfeepercent / 2, best-fee * 2).  i.e. if we're well under our `maxfeepercent`, then you can spend twice as much as the best case on route obfuscation.

Claudia Diaz made some pointed remarks about our route selection in Lightning, and I'd really like to improve it.

Anyway, this is an aside: I'd prefer to avoid exposing this for now.

Reply:
Ok, will remove this commit then.

Reply:
> I'd be happier with moving the routing algo towards randomizing the route so that it spends up to min(maxfeepercent / 2, best-fee * 2).

Is this part of the reason why you proposed higher fees, to have more chances to fuzz route computation ?